src/storage/conversion/ConversionUtil.ts

```diff
@@ -57,6 +56,29 @@ RepresentationPreference[] => {
   return weightedSupported.filter((preference): boolean => preference.weight !== 0);
 };
 
+/**
+ * Checks if the given two media types/ranges match each other.
+ * Takes wildcards into account.
+ * @param mediaA - Media type to match.
+ * @param mediaB - Media type to match.
+ *
+ * @returns True if the media type patterns can match each other.
+ */
+export const matchingMediaType = (mediaA: string, mediaB: string): boolean => {
+  const [ typeA, subTypeA ] = mediaA.split('/');
```

Comment:
Theoretically, there is the case `*/*/a` or any arbitrary number of slashes. I wonder if we want regexes instead.

Reply:
That would not be a valid Accept header though. Would have to double check for content-types but I would assume it's also illegal there?

Reply:
Fair enough, as long as we're aware.